src/zope/testing/setupstack.txt

```diff
@@ -228,6 +228,7 @@ Here's an example:
     ...
     ...     def setUp(self):
     ...         self.setUpDirectory()
+    ...         os.mkdir('example')
```

Comment:
That is the point of this?

Reply:
This is needed to get coverage for: https://github.com/zopefoundation/zope.testing/blob/8a46f1901bcaa871b919fc1ec2f22abd86587e94/src/zope/testing/setupstack.py#L66-L68

Reply:
Maybe add a comment?

Reply:
Done in https://github.com/zopefoundation/zope.testing/pull/38/commits/a6490fd98af64bbf1eb4706347782ccb98f025df